src/index.html

```diff
@@ -24,11 +24,18 @@
 
       <main class="content">
         <section class="profile page__section">
-          <img
-            src="<%=require('./images/Justin-delaflor.jpg')%>"
-            alt="profile-image"
-            class="profile__avatar"
-          />
+          <div class="profile__image-container">
+            <img
```

Comment:
Needs correction: when you hover the profile picture the pen icon should appear, not the default avatar (see the design)